scripts/util.py

```diff
@@ -1716,20 +1683,6 @@ def instance(self, instance_name, project=None, slurm_cluster_name=None):
         )
         return instances.get(instance_name)
 
-    def describe_instance(self, instance_name, project=None, zone=None):
```

Comment:
https://github.com/GoogleCloudPlatform/slurm-gcp/blob/master/test/test_nodes.py#L75 Is it referenced ?